devops/sql/data-load-scripts/data load -Accounts & Transactions.ipynb

```diff
@@ -0,0 +1,1503 @@
+{
```

Comment:
Please use camel case or - in file names. No empty spaces, lets keep it uniform.